app/src/main/java/com/example/makeitso/model/service/Performance.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.makeitso.model.service
```

Comment:
Can you add the copyright to the beginning of the file? (It was mandatory when I submitted the codebase for review to publish under `FirebaseExtended`).